python/cuspatial/CMakeLists.txt

```diff
@@ -44,40 +42,22 @@ endif()
 if(NOT cuspatial_FOUND)
   set(BUILD_TESTS OFF)
   set(BUILD_BENCHMARKS OFF)
-  set(_exclude_from_all "")
-  if(CUSPATIAL_BUILD_WHEELS)
-
-    # Statically link cudart if building wheels
-    set(CUDA_STATIC_RUNTIME ON)
-    set(CUSPATIAL_USE_CUDF_STATIC ON)
-    set(CUSPATIAL_EXCLUDE_CUDF_FROM_ALL ON)
-
-    # Always build wheels against the pyarrow libarrow.
-    set(USE_LIBARROW_FROM_PYARROW ON)
```

Comment:
@vyasr This causes cuDF's CMake to [build and install a second libarrow](https://github.com/rapidsai/cuspatial/actions/runs/7118401836/job/19381328724#step:9:719). Don't we need to link to pyarrow's `libarrow.so` due to the internal globals?

Reply:
Yes, this was an error on my part. I went through a couple of iterations of updating the cudf CMake and at one point I had made this detection automatic within cudf. However the other unifications here are desirable, so we just want to specify that variable. Fixed in #1334.